sw/airborne/subsystems/ahrs/ahrs_int_cmpl_quat.c

```diff
@@ -270,7 +270,11 @@ void ahrs_icq_update_accel(struct Int32Vect3 *accel, float dt)
 #define ACC_FROM_CROSS_FRAC INT32_RATE_FRAC + INT32_SPEED_FRAC - INT32_ACCEL_FRAC - COMPUTATION_FRAC
 
     const struct Int32Vect3 vel_tangential_body =
+#ifdef VERTICAL_WING_HYBRID
+      {0, 0, -(ahrs_icq.ltp_vel_norm >> COMPUTATION_FRAC)};
```

Comment:
Yes, it would be very good to make this more generic.

 - one option is to use the 3D gps speed (this would change the behaviour of the filter)
 - another option is to define the FIXEDWING_FRONT_AXIS
 - another option is to define the IMU2FIXEDWING_ROTATION

Any preference